### ISSUE TITLE
DnD issue about remove node from dom.

### ISSUE THREAD

**Issue by contact2mayurkukadiya** [kutlugsahin/ngx-smooth-dnd#38]
I am using this package into angular project. I found one error that mention below. After this error occur my DnD not working. can't drag and nothing happen. I don't know why this error raised and how to solve. 

![Capture](https://user-images.githubusercontent.com/41474341/67358796-198f8a80-f57f-11e9-941a-8b3dccfad5ae.PNG)

When I drag one item from lot's of item list (20 items), then automatically this error occur.

**Comment by contact2mayurkukadiya** [comment]
Is there any way to prevent adding ghost element in  DnD container ?

**Comment by umeshpatadiya** [comment]
getting same error :(

**Comment by sarahm7** [comment]
yes same issue here. this usually happens if you drag back and forth between containers.

**Comment by contact2mayurkukadiya** [comment]
@sarahm7  : but I am using only one container. I am using this package for sort array. and still this error encountered.
